src/app/components/login/login-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
```

Comment:
why do we need this file? we already have the /booking/flight route defined in the app module.